### PR TITLE
[ISSUE #641]Rename NameSrvAddrList to NameServerAddressList in OPS Page

### PR DESCRIPTION
**Pull request by fantow**
fix the [issue#641](https://github.com/apache/rocketmq-externals/issues/641)

### CODE REVIEW

File: rocketmq-console/src/main/resources/static/view/pages/ops.html [hunk 1]
```diff
@@ -16,23 +16,26 @@
...
         <h2 class="md-title">IsUseVIPChannel</h2>
-        <div  class="pull-left">
+        <div class="pull-left">
             <md-switch class="md-primary" md-no-ink aria-label="Switch No Ink" ng-model="useVIPChannel">
             </md-switch>
         </div>
         <div class="pull-left">
-            <button class="btn btn-raised btn-sm btn-primary" type="button" ng-click="updateIsVIPChannel()">{{'UPDATE' | translate}}
+            <button class="btn btn-raised btn-sm btn-primary" type="button" ng-click="updateIsVIPChannel()">{{'UPDATE' |
+                translate}}
```
Comment (vongosling) [inline rocketmq-console/src/main/resources/static/view/pages/ops.html:38]: Do not format here !

File: rocketmq-console/src/main/resources/static/view/pages/ops.html [hunk 2]
```diff
@@ -16,23 +16,26 @@
   -->
 <div class="container-fluid" id="deployHistoryList">
     <div class="page-content">
-        <h2 class="md-title">NameSvrAddrList</h2>
+        <h2 class="md-title">NameServerAddressList</h2>
         <div class="pull-left">
-            <md-chips ng-model="namesvrAddrList"  readonly="ctrl.readonly" md-removable="ctrl.removable" md-enable-chip-edit="true"></md-chips>
+            <md-chips ng-model="namesvrAddrList" readonly="ctrl.readonly" md-removable="ctrl.removable"
```
Comment (vongosling) [inline rocketmq-console/src/main/resources/static/view/pages/ops.html:21]: Like others, you could format only the update spot.

File: rocketmq-console/src/main/resources/static/view/pages/ops.html [hunk 3]
```diff
@@ -16,23 +16,26 @@
...
+                      md-enable-chip-edit="true"></md-chips>
         </div>
         <div class="pull-left">
-            <button class="btn btn-raised btn-sm btn-primary" type="button" ng-click="updateNameSvrAddr()">{{'UPDATE' | translate}}
+            <button class="btn btn-raised btn-sm btn-primary" type="button" ng-click="updateNameSvrAddr()">{{'UPDATE' |
```
Comment (vongosling) [inline rocketmq-console/src/main/resources/static/view/pages/ops.html:25]: Why format here ?

Comment (fantow) [reply rocketmq-console/src/main/resources/static/view/pages/ops.html:25]: Sry,I sumbit another PR to fix it.I will close this.

**Pull request closed without merging**